src/utils/rest-api.js

```diff
@@ -348,6 +348,28 @@ export function createStudy(
     }
 }
 
+export function createCase({ name, description, file, parentDirectoryUuid }) {
+    console.info('Creating a new case...');
+    let urlSearchParams = new URLSearchParams();
+    urlSearchParams.append('description', description);
```

Comment:
We may call `encodeURIComponent` over `description` at least I think.

Reply:
done